pkg/crc/preflight/preflight_darwin.go

```diff
@@ -10,6 +10,13 @@ import (
 // SetupHost performs the prerequisite checks and setups the host to run the cluster
 func hyperkitPreflightChecks(networkMode network.Mode) []Check {
 	return []Check{
+		{
+			configKeySuffix:  "check-supported-cpu-arch",
+			checkDescription: "Checking if running on a supported CPU architecture",
+			check:            checkSupportedCPUArch,
+			fixDescription:   "CodeReady Containers is unsupported on Apple M1 hardware",
```

Comment:
@cfergeau Since `checkSupportedCPUArch` handle M1 and also any other arch so in `fixDescription` just mentioning `Apple M1` would be enough?

Reply:
Not sure what you mean